### PR TITLE
Main navigation page

### PR DESCRIPTION
**Pull request by GabriellaMSanchez**
Reconfigured MainNavPage text and component to match the Mock up better. Removed quote from footer and created real footer. Added ButtonGroup, reconfigured main-nav-button and search-bar CSS to match mock up. Moved search-bar so it was more intuitive for the user.
Screenshot of changes below:
Screen Shot 2023-10-02 at 6 16 19 PM

### CODE REVIEW

**Comment by GabriellaMSanchez** [comment]
Changes to searchBar

Screen Shot 2023-10-02 at 7 00 27 PM

**Comment by GabriellaMSanchez** [comment]
Button img and text alignment

Screen Shot 2023-10-02 at 7 07 29 PM

**Comment by DanielRyanSmith** [comment]
It looks like a number of merge conflicts have to be dealt with before I can take a look at this 🙂

**Comment by GabriellaMSanchez** [comment]
It should be ready now! :)

File: src/SearchBar.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-        <input
+        <input className="search-bar"
         type="text" 
         placeholder="Search a Park"
-        style={{border: '3px solid black'}}
+        style= {{
+            height: "100%",
+            width: "100%",
+            paddingLeft: "8px"
+        }}
```
Comment (DanielRyanSmith) [inline src/SearchBar.js:11]: Style should be moved to a .css file, and give this element an identifier

File: src/SearchBar.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
 
 function SearchBar() {
     return (
-        <input
+        <input className="search-bar"
         type="text" 
         placeholder="Search a Park"
```
Comment (DanielRyanSmith) [inline src/SearchBar.js:6]: more indentation here

**Review by DanielRyanSmith — APPROVED** [review]
Approved after some minor fixes. Good work!

File: src/SearchBar.js (hunk 1 above)
Comment (GabriellaMSanchez) [reply src/SearchBar.js:11]: Fixed!

File: src/SearchBar.js (hunk 2 above)
Comment (GabriellaMSanchez) [reply src/SearchBar.js:6]: Fixed!

**Comment by DanielRyanSmith** [comment]
I already approved this, so you're free to merge it after you took care of the small nits

**Pull request merged**